LondonBritishLibrary/orient/BLorient487.xml

```diff
@@ -158,28 +165,24 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                     </desc>
                                     <q xml:lang="gez">ነገሥት፡ ዘቅ' ፡ መ' ፡ ዓለም።</q>
                                 </item>
-                                
-                                
-                                
-                                
                             </list>
                         </additions>
                         
-                        
-                        
                         <bindingDesc>
-                            <binding contemporary="true" xml:id="binding">
-                                
-                                <decoNote xml:id="b0">In the original boards. </decoNote>
+                            <binding contemporary="partly" xml:id="binding">
+                                <decoNote xml:id="b0">Wooden boards with leather quarter cover. According to Wright, the boards are original.
+                                    While this cannot be excluded, the back board looks very different from the front board and might be a
+                                substitution by the library. The leather quarter cover also does not seem to be original, but to have been added by the library, 
```

Comment:
This is very assumptive and at the end the use is lost. 
(After Wright a lot of time passed, at his time both boards could be original. We describe what we see. I assume many mss changed their appearance after having been catalogued, but we normally do not compare ).

"Wooden boards with leather quarter cover. The back board and the leather cover are unlikely original, 
probably added in the Library."

Reply:
I agree with the proposed formulation, but I do think that the difference from the state of the manuscript at the time of Wright's description is an important information, which would be lost on those who would not open the catalogue in parallel (to make that a necessity for reading our descriptions would undermine our project a bit, I think).
How about something like:
"Wooden boards with leather quarter cover. The back board and the leather cover are unlikely original, probably added in the Library. Wright states that the boards are original."

Reply:
It is ok, buy Wright seems to have done this very irregularly, isnt it? And it is unclear why in this case he says this. 
 The difficulty that i I see is that, if you say "Wright states...", you need a couple of arguments as to why what he says is wrong. He had the book in his hands, you did not. Perhaps, you could style it like:  "...Library (Wright, on the contrary, states that both boards are original)."

Reply:
All together, could be like this: 
Wooden boards with leather quarter cover. The back board looks very different from the front board and might have been furnished in the Library (Wright, on the contrary, states that both boards are original). The leather quarter cover also does not seem to be original, possibly supplied together with the back board.

Reply:
My hypothesis is that he stated this for the original bindings and those he stays silent on were rebound already at his time, but it is really not very certain. I think your solution is good.